rails-questions.md

```diff
@@ -2,33 +2,57 @@
 
 1. What is the command to create a new Rails application with a Postgres database?
 
-
+  * rails new i_feel_confident_in_this_one -d postgresql -T and then we must cd into i_feel_confident_in_this_one and then put rails db:create and open that bad boy up and have some fun!
 
 2. Explain each section of the following route:  get '/library/:id' => 'book#show'
 
-get -
-library -
-:id -
-book -
-show -
+get - get is pretty much what it says. It is getting the key/value/etc from a specific place in your database (through the route).
+
+library - I believe that library is the class. Like this?
+  class Library < ApplicationRecord
+    has_many :books
+  end
+
+:id - this is the key or value (I know they are different I think this one is the value? I have just heard tutorials call them different things hahaha. (I might be getting my terminology wrong so im sorry, I do need to memorize some of those cuz that will mess me up.)
+
+book - this would be the "MainController" for the examples we used last. I am trying to remember the actual term. I believe it would be whatever you do like this? 
+rails g controller book ? I think hahaha
+
+show - I believe this is the method name we are using? So like for this one it would be:
+  def show
+    @show (then whatever would complete what its trying to do) 
+  end
 
 
 3. Oops, I forgot to add a foreign key to my model. Describe the steps to remedy this mistake.
 
+  * You would want to use CREATE INDEX to index them and then you would want to use ADD CONSTRAINT statement to add the FOREIGN KEY constraint to the columns.
```

Comment:
look at the section in the syllabus and watch Andee's lecture on rails migrations for this one